src/In.ProjectEKA.HipService/Discovery/Database/Migrations/20200706065543_MaskedPatientReferenceNumber.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace In.ProjectEKA.HipService.Discovery.Database.Migrations
+{
+    public partial class MaskedPatientReferenceNumber : Migration
```

Comment:
Codacy found an issue: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5811615)